server/services/participants.js

```diff
@@ -612,7 +612,46 @@ const getParticipantByIdWithStatus = async ({ id, userId }) =>
     })
     .find({ id, 'user.user_id': userId });
 
+const setParticipantLastUpdated = async (id) => {
+  // Find participants
+  let participant = (await getParticipantByID({ id }))[0];
```

Comment:
```suggestion
  let [participant] = await getParticipantByID({ id });
```
nitpick: I like this format more for getting an entry in an array

Reply:
10/10 Looks way better.